### ISSUE TITLE
[Bug]: Privacy Manifest Issue

### ISSUE THREAD

**Issue by bejewel-daye** [braze-inc/braze-swift-sdk#125]
### Platform

iOS

### Platform Version

All

### Braze SDK Version

9.0.0

### Xcode Version

Xcode 15.2

### Computer Processor

Apple (M1)

### Repro Rate

100%

### Steps To Reproduce

- 

### Expected Behavior

- 

### Actual Incorrect Behavior

- 

### Additional Information

I updated the Braze to 9.0.0 but still have an issue.
I got an email from Apple when we submit our app:

> ITMS-91053: Missing API declaration - Your app’s code in the “PlugIns/NotificationContentExtension.appex/NotificationContentExtension” file references one or more APIs that require reasons, including the following API categories: NSPrivacyAccessedAPICategoryUserDefaults. While no action is required at this time, starting May 1, 2024, when you upload a new app or app update, you must include a NSPrivacyAccessedAPITypes array in your app’s privacy manifest to provide approved reasons for these APIs used by your app’s code. For more details about this policy, including a list of required reason APIs and approved reasons for usage, visit: https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api.
> 
> Apple Developer Relations

**Comment by lowip** [comment]
Hi @bejewel-daye,

Thank you for reporting this issue. Could you provide more information regarding how you are currently integrating the SDK, specifically:
- Integration method (SPM, CocoaPods, manual)
- Repository (see [alternative repositories](https://github.com/braze-inc/braze-swift-sdk#alternative-repositories))

**Comment by bejewel-daye** [comment]
Hi @lowip,
These are the details:

- Integration method
  - SPM
- Repository (I'm not sure it is correct information for you)
  - [braze-inc/braze-swift-sdk](https://github.com/braze-inc/braze-swift-sdk)

**Comment by lowip** [comment]
Thank you for these information.

We're still investigating for a possible solution on our side. In the meantime, you should be able to resolve the issue on your own by following the workaround in this [comment](https://github.com/braze-inc/braze-swift-sdk/issues/121#:~:text=We%20are%20actively%20looking%20into%20this%2C%20but%20as%20a%20workaround%20in%20the%20meantime%2C%20you%20can%20manually%20copy%20the%20API%20declarations%20we%20have%20listed%20in%20our%20published%20privacy%20manifests%20into%20your%20own%20app%2Dlevel%20manifests.%20You%20will%20need%20to%20do%20this%20for%20any%20application%20target%20that%20imports%20either%20BrazeKit%20or%20BrazePushStory.).

We will make sure to update this thread with any new findings.

Best,

**Comment by bejewel-daye** [comment]
Hi @lowip,
Is there any update about this issue?
As May 1st is approaching, I'd like to know if there's an update schedule or any information for this issue.

Thanks,

**Comment by hokstuff** [comment]
Hi @bejewel-daye,

We are attempting to reproduce the issue and have a few follow up questions for you:
1. When you receive this error, which of the [Braze packages](https://github.com/braze-inc/braze-swift-sdk/blob/22a9c2d49bf1f953af9f54228736d0f73ba5904b/Package.swift#L14) did you import in your app (e.g. BrazeKit, BrazeUI, BrazePushStory, BrazeNotificationService, etc)?
2. Have you been able to attempt [the workaround described here](https://github.com/braze-inc/braze-swift-sdk/issues/125#issuecomment-2079617127), and does it remove those warnings during upload?

Thanks!

**Comment by bejewel-daye** [comment]
Hi @hokstuff, 

1. We've been using BrazeKit, BrazeUI, BrazePushStory, BrazeNotificationService. I couldn't tell you specific package. 
2. I attached the warning message on the section of 'Additional Information'. I think It looks being with my app target.
3. Before updating the Braze SDK to version 9.0.0, I encountered the same warning message. This is the only remaining warning message.

Thanks.
